.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: 'CI'
+
+on:
+  pull_request:
+    paths-ignore:
+      - 'docs/**'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [10, 12, 14]
```

Comment:
Do we still need to check v10?

Reply:
I'm doing some testing for now but as soon as it is in maintenance we can remove it

Reply:
It's in Maintenance status now:

![image](https://user-images.githubusercontent.com/8593673/95368939-9e0c1380-088b-11eb-9834-2636a20b7e92.png)